test/public/runs/overview.test.js

```diff
@@ -88,6 +90,7 @@ module.exports = () => {
     });
 
     it('shows correct datatypes in respective columns', async () => {
+        await goToPage(page, 'run-overview');
```

Comment:
We should avoid as much as possible to use this. You are already in this page in the test before, why using `goToPage` here?

Reply:
Removed this one, but George mentioned that this was used to reset state of the page so I don't think it's a good idea to remove this everywhere right now